engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbJobState.java

```diff
@@ -144,7 +144,7 @@ public void timeout(final long key, final JobRecord record) {
     validateParameters(type);
     EnsureUtil.ensureGreaterThan("deadline", deadline, 0);
 
-    createJob(key, record, type);
+    updateJob(key, record, State.ACTIVATABLE);
```

Comment:
:question: Please explain this change. This seems to be different than the others and it is not clear to me, why this is a valid change or even a necessary one.